### ISSUE TITLE
Start in the Background button doesn't do anything

### ISSUE THREAD

**Issue by Insane-Owl** [mcserversoft/IssueTracker#13]
**Describe the bug**
The "Start in the background" doesn't do anything

**To Reproduce**
1. Go to Options > General > Automatically start on system startup > Start in the Background and turn it on.
2. Restart your PC.
3. MCSS will be start into it's open state, and not to the system tray.

**Expected behavior**
I expect MCSS to start to the system tray, not open.

**Screenshots**
![Screenshot 2024-01-21 133123](https://github.com/mcserversoft/IssueTracker/assets/78328941/51a305f6-61f3-4897-9e6c-bc2a6fbdbd9c)

**Desktop (please complete the following information):**
 - OS Version: Windows 11
 - MCSS 13.8.0

**Comment by Fiahblade** [comment]
Hi, thanks for reporting the issue. What version of Windows 11 are you using?

**Comment by Insane-Owl** [comment]
Windows 11 Pro 22635.3858

**Comment by Insane-Owl** [comment]
idk if this is related, but MCSS is also always in the alt+tab menu, even when minimized to tray
![image](https://github.com/user-attachments/assets/2b78e605-6d9c-4205-80f1-5f3f89f168eb)

**Comment by Fiahblade** [comment]
Yeah, this needs an overhaul/closer look. Thanks for reporting the issue.
